.github/workflows/coverage.yml

```diff
@@ -18,12 +18,14 @@ jobs:
     if: "!contains(github.event.head_commit.message, '[skip ci]')"
 
     steps:
-    - name: Build Gwion
-      uses: fennecdjay/gwion-action@v1
-      with:
-        dir: .
-        ref: ${{ github.sha }}
-        run: 'true'
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Init submodules
+      run: git submodule update --init util ast
+
+    - name: Build & Test Gwion
+      run: make && make test
```

Comment:
It might be better to have separate test for build and commit.
I know gwion-action did all at once, but it was/is a design error :confused:

Just to make it clear, only the build part needs `USE_COVERAGE` set.

Reply:
Do you mean, the build and test should be separate steps?